foyer/forcefield.py

```diff
@@ -947,5 +967,418 @@ def _write_references_to_file(self, atom_types, references_file):
                 bibtex_text = bibtex_text[:-2] + note + bibtex_text[-2:]
                 f.write('{}\n'.format(bibtex_text))
 
+    def get_parameters(self, group, key, keys_are_atom_classes=False):
+        """Get parameters for a specific group of Forces in this Forcefield
+
+        Parameters
+        ----------
+        group: str
+            One of {"atoms", "bonds", "angles", "periodic_propers", "periodic_impropers", "rb_propers", "rb_impropers"}.
```

Comment:
Since "bonds" and "angles" have multiple types of Forces in OpenMM, it might be nice to be more specific with the names. This will allow us to add additional/specific functions for the parameters extraction,